Concept_Templates/single_crystal.md

```diff
@@ -0,0 +1,36 @@
+## Knowledge domain resources
+
+### Related
+
+https://dictionary.iucr.org/crystal: historical, reciprocal and direct space definitions. 
+Expert discussions also raised the issue of different dimensionality crystals, quasicrystals and so on.
+
+https://en.wikipedia.org/wiki/Crystal: information as above, as well as history of the term,
+some discussion of various types of crystal including quasicrystals and crystal defects.
+
```

Comment:
Just so it doesn't disappear in my mail pile: @sauliusg suggests this is also a good reference:
```suggestion
https://doi.org/10.1080/0889311X.2021.1978080: several definitions of the concept crystal and the evolution of the term over the last 100 years.
```